tests/unit_tests/scheduler/test_job.py

```diff
@@ -180,6 +182,28 @@ async def test_num_cpu_is_propagated_to_driver(realization: Realization):
         name=realization.run_arg.job_name,
         runpath=Path(realization.run_arg.runpath),
         num_cpu=8,
+        realization_memory=0,
+    )
+
+
+@pytest.mark.usefixtures("use_tmpdir")
+@pytest.mark.asyncio
+async def test_realization_memory_is_propagated_to_driver(realization: Realization):
+    realization.realization_memory = 8 * 1024**2
+    scheduler = create_scheduler()
+    job = Job(scheduler, realization)
+    job_run_task = asyncio.create_task(job.run(asyncio.Semaphore(), max_submit=1))
+    job.started.set()
+    job.returncode.set_result(0)
+    await job_run_task
+    scheduler.driver.submit.assert_called_with(
+        realization.iens,
+        realization.job_script,
+        realization.run_arg.runpath,
+        num_cpu=1,
+        realization_memory=8 * 1024**2,
```

Comment:
Can we put the magic number `8*1024**2` in a variable and use it on line 192 too?

Reply:
fixed.